src/js/angular/core/directives/copy-to-clipboard/copy-to-clipboard.directive.js

```diff
@@ -41,42 +59,100 @@ copyToClipboard.$inject = ['$translate', 'toastr'];
  */
 function copyToClipboard($translate, toastr) {
     return {
-        // Note: the line-height of the element must match the line-height of the icon
         template: `
             <style>
                 .copy-btn {
                     line-height: 0.75;
+                    position: relative;
+                }
+
+                .custom-link-icon {
+                    opacity: 0;
+                    margin-left: 5px;
+                    transition: opacity 0.2s ease-in-out;
+                    color: var(--secondary-color);
+                }
+
+                .copy-btn:hover .custom-link-icon {
+                    opacity: 1;
                 }
             </style>
-            <button class="btn btn-link btn-sm copy-btn" gdb-tooltip="{{tooltipText | translate}}" ng-click="copyToClipboard()"><i class="fa fa-clone"></i></button>
+            <button
+                class="btn btn-link btn-sm copy-btn"
+                ng-if="!customTooltipStyle"
+                gdb-tooltip="{{tooltipText | translate}}"
+                ng-click="copyToClipboard($event)">
+                <i class="fa fa-clone link-icon" aria-hidden="true"></i>
+            </button>
+            <button
+                class="btn btn-link btn-sm copy-btn"
+                ng-if="customTooltipStyle"
+                ng-click="copyToClipboard($event)">
+                <i class="fa fa-clone custom-link-icon" aria-hidden="true"></i>
+            </button>
         `,
         restrict: 'E',
         scope: {
             tooltipText: '@',
-            textToCopy: '@'
+            textToCopy: '@',
+            customTooltipStyle: '@?',
+            targetSelector: '@?'
         },
         link: function ($scope, element) {
             $scope.copyToClipboard = function() {
                 const textToCopy = $scope.textToCopy ? $scope.textToCopy : element.parent().find('.copyable').text();
 
-                if (navigator.clipboard) {
-                    navigator.clipboard.writeText(textToCopy).then(() => {
-                        toastr.success($translate.instant('import.help.messages.copied_to_clipboard'));
-                    }).catch((err) => {
-                        console.error('Could not copy text: ', err);
+                function showCustomTooltip() {
+                    const tooltip = document.createElement("span");
+                    tooltip.innerText = $translate.instant('common.copied');
```

Comment:
This is not OK, because this directive is generic and can be used for copying not only IRI's, but any text. So, better consider providing the tooltip key as parameter maybe